src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -304,5 +304,67 @@ public void Dispose()
                 }
             }
         }
+
+        [Fact]
+        public static void VerifyHashSignature_SHA256_2048()
```

Comment:
Looks like a lot of duplication with https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/SignVerify.cs#L676.  What are the differences, and could anything be shared between them?